components/builder-web/stylesheets/base/_forms.scss

```diff
@@ -136,3 +136,15 @@ select {
   -webkit-appearance: none;
   width: auto;
 }
+
+::placeholder {
+  color: $hab-gray;
+}
+
+// Angular Material button overrides
+
+.mat-fab:not([class*=mat-elevation-z]),
+.mat-mini-fab:not([class*=mat-elevation-z]),
+.mat-raised-button:not([class*=mat-elevation-z]) {
+    box-shadow: none !important;
+}
```

Comment:
Removes the dark gray shadow on all the buttons to more closely reflect the button design in mockups.

Reply:
![tenor-82911830](https://user-images.githubusercontent.com/1130349/31912166-a8f27040-b808-11e7-9b34-f16d000ca669.gif) not my hover buttons!